### PR TITLE
enable multiple binaries

### PR DESCRIPTION
**Pull request by danbi2990**
Hi. Thank you for this nice project.
Could you check if this PR is acceptable?
It's for making multiple binaries (e.g. encode & decode) in a single workflow.

### CODE REVIEW

File: cli/src/main.rs [hunk 1]
```diff
@@ -83,18 +83,22 @@ fn build(release: bool) -> Result<()> {
...
+    fs::create_dir_all(&workflow_dir)?;
 
-    let src = target_dir.join(mode.dir()).join(&binary_name);
-    let dst = workspace_dir.join("workflow").join(&binary_name);
-    fs::copy(&src, &dst)?;
+    binary_names.iter().for_each(|binary_name| {
+        let src = target_dir.join(mode.dir()).join(binary_name);
+        let dst = workflow_dir.join(binary_name);
+        fs::copy(&src, &dst).unwrap();
+    });
```
Comment (rossmacarthur) [inline cli/src/main.rs]: I would prefer a normal `for` loop here and then you can bubble up the io error

```suggestion
    for binary_name in &binary_names {
        let src = target_dir.join(mode.dir()).join(binary_name);
        let dst = workflow_dir.join(binary_name);
        fs::copy(&src, &dst)?;
    }
```

File: cli/src/main.rs [hunk 2]
```diff
@@ -83,18 +83,22 @@ fn build(release: bool) -> Result<()> {
...
 
     print(
         "Copied",
         format!(
-            "binary to `{}`",
-            dst.strip_prefix(env::current_dir()?)?.display()
+            "binary to {}/{{{}}}",
+            workflow_dir.strip_prefix(env::current_dir()?)?.display(),
+            binary_names.join(", ")
```
Comment (rossmacarthur) [inline cli/src/main.rs]: Perhaps just put whole print in the loop above?

**Review by rossmacarthur — CHANGES_REQUESTED** [review]
Thanks for the contribution 😄, just some minor comments.

**Comment by danbi2990** [comment]
The changes have been applied as you pointed out. Thanks again.

**Review by rossmacarthur — APPROVED** [review]

**Pull request merged**